ts/screens/preferences/ServiceDetailsScreen.tsx

```diff
@@ -91,6 +91,23 @@ function renderInformationRow(
   );
 }
 
+function renderTruncatedInformationRow(
```

Comment:
what about just truncating every text field? why not just add `numberOfLines={1}` to  `renderInformationRow` ?